ckanext/canada/templates/internal/recombinant/resource_edit.html

```diff
@@ -18,7 +18,7 @@
     <div class="panel panel-primary">
       <div class="panel-heading">{{ _("Guide") }}</div>
       <div class="panel-body">
-        <p>{% trans -%}For information on how to prepare, maintain and upload templates to the Registry, refer to the appropriate documentation posted on <a href="http://www.gcpedia.gc.ca/wiki/Proactive_Disclosure_on_Open.Canada.ca">GCPEDIA</a>{% endtrans %}</p>
+        <p>{{ _('For information on how to prepare, maintain and upload templates to the Registry, refer to the appropriate documentation posted on {link}').format(link=_('<a href="http://www.gcpedia.gc.ca/wiki/Open_Data_Inventory_/_l%E2%80%99inventaire_de_donn%C3%A9es_ouvertes">GCPEDIA</a>'))|safe }}</p>
```

Comment:
Not sure what that would involve so I will remove this change from this PR and we can fix it later.